permissions/Compatibility.permission

```diff
@@ -0,0 +1,41 @@
+# -*- mode: sh -*-
+# This provides compatibility for older apps
+# The permission is not shown to user,
+# thus it ***must not*** give access to additional data or hardware resources
+
+# x-sailjail-translation-catalog =
+# x-sailjail-translation-key-description = permission-la-compatibility
+# x-sailjail-description = Compatibility
+# x-sailjail-translation-key-long-description = permission-la-compatibility_description
+# x-sailjail-long-description =
+
+# PDF tools
```

Comment:
Nice!
I don't want to be presumptuous... but i presume this is mostly for SeaPrint?
In that case i can tell you i only use pdftoppm, pdftops, pdftocairo, pdfinfo currently (SeaPrint 0.9).
For 1.0 i am dropping the use of all poppler-utils binaries, so i can't see that list growing at all.
But this is really a good respite from having to get that completely sorted with a deadline approaching.

Reply:
Yes. I don't know anything else that would be using these tools. I included everything initially but the idea was to reduce them before merging depending on what's really needed. Thanks for telling me know which tools you actually use, I appreciate that!